web/static/js/components/timezones/TimezoneAutocomplete.jsx

```diff
@@ -66,8 +67,14 @@ class TimezoneAutocomplete extends Component {
     dispatch(actions.setTimezone(timezone.id))
   }
 
+  getTimeForTimezone(timezone, locale) {
+    var date = new Date(Date.now())
```

Comment:
```suggestion
    var date = new Date()
```

this already gives the current local time.